articles/active-directory/saas-apps/ms-confluence-jira-plugin-adminguide.md

```diff
@@ -59,23 +59,12 @@ Note the following information before you install the plug-in:
 
 The plug-in supports the following versions of Jira and Confluence:
 
-* Jira Core and Software: 6.0 to 7.12
+* Jira Core and Software: 6.0 to 8.22.1
 * Jira Service Desk: 3.0.0 to 3.5.0
```

Comment:
We will need to update the JIRA Service Desk 3.0 to 4.22.1. Refer the doc page : https://docs.microsoft.com/en-us/azure/active-directory/saas-apps/jiramicrosoft-tutorial